### PR TITLE
LPS-191885 - Fix Automation | Data Engine | Master

### PR DESCRIPTION
**Pull request by thaiscabrals**
**Ticket:** 

- [LPS-191885](https://liferay.atlassian.net/browse/LPS-191885)

Tests:
-  DEDateField#CanPickDateUsingCalendarWidget
- RichTextField#PublishWebContentWithEmptyRequiredRichText

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DEDateField.testcase [hunk 1]
```diff
@@ -289,7 +289,9 @@ definition {
 			valueMonth = "August",
 			valueYear = 2021);
 
-		Click(locator1 = "Button#PUBLISH");
+		Click(
+			key_text = "Publish",
+			locator1 = "Button#ANY");
```
Comment (thaiscabrals) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DEDateField.testcase:294]: The publish button has changed.
The tests have been tried and the result is in the ticket.

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/RichTextField.testcase [hunk 2]
```diff
@@ -194,7 +194,9 @@ definition {
 			structureName = "WC Structure Name",
 			webContentTitle = "WebContent Title");
 
-		Click(locator1 = "Button#PUBLISH");
+		Click(
+			key_text = "Publish",
+			locator1 = "Button#ANY");
```
Comment (thaiscabrals) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/RichTextField.testcase:199]: The publish button has changed.
The tests have been tried and the result is in the ticket.

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**